data/json/npcs/TALK_EVAC_GUARD1.json

```
@@ -72,9 +72,10 @@
   {
     "type": "talk_topic",
     "id": "TALK_EVAC_GUARD1_JOB",
-    "dynamic_line": "Uh, not really.  Go talk to a merchant if you have anything to sell.  Otherwise the Old Guard liaison might have something, if you can find him.",
+    "dynamic_line": "Depends on what you want. Go talk to a merchant if you have anything to sell.  Otherwise the Old Guard liaison might have something, if you can find him. But if you're just looking for someone to put a good word in, I might have something for you.",
```

Style is two spaces after a sentence:
```suggestion
    "dynamic_line": "Depends on what you want.  Go talk to a merchant if you have anything to sell.  Otherwise the Old Guard liaison might have something, if you can find him.  But if you're just looking for someone to put a good word in, I might have something for you.",
```